graph/src/data/subgraph/api_version.rs

```diff
@@ -55,10 +55,12 @@ pub const SPEC_VERSION_1_1_0: Version = Version::new(1, 1, 0);
 pub const SPEC_VERSION_1_2_0: Version = Version::new(1, 2, 0);
 
 // Enables subgraphs as datasource
+// Change the way the VID field is generated. It used to be autoincrement. Now its
+// based on block number and the sequence of the entities in a block.
```

Comment:
This should also mention that ordering by `vid` now, across all entity types for a subgraph, returns entities in the order in which they were written.